cluster/log-dump/log-dump.sh

```diff
@@ -675,6 +690,9 @@ function main() {
   fi
 
   detect_node_failures
+  if [[ ${log_dump_expected_success_percentage} -gt 0 && ${logexporter_failed} -ne 0 ]]; then
```

Comment:
nit: I would reverse conditions to be more readable

Reply:
Tbh, the current order sounds more natural to me (`if failing_on_unsuccessful_log_dumping_enabled && logexporter_failed`) but I might be slightly biased here because I wrote that logic. Nevertheless, I'll switch the conditions if their sequence sounds more natural.

Reply:
Done.